### DISCUSSION TITLE
I don t need the id beginning # id

### DISCUSSION THREAD

**Discussion by zeMinng** [ai/nanoid#489] [Q&A]
I don t need the id beginning # id. Because it will bring unsatisfactory results

![image](https://github.com/user-attachments/assets/86365944-273d-4047-8d62-e240abc021f6)

**Comment by ai** [comment]
Nano ID doesn't use `#` in the default alphabet.

The issue seems like is in the your code (or other library) using Nano ID.

**Reply by zeMinng** [reply]
oh yes. I know why, and js dom id should not be available for numbers

**Reply by ai** [reply]
To avoid numbers just add prefix with letters: `"id" + nanoid()`

**Reply by zeMinng** [reply]
thanks
